.travis.yml

```diff
@@ -87,3 +87,7 @@ deploy:
 env:
   global:
     secure: nRtG4v+jNQbCQg/qhYwrPaBMuwGxG8z2+kfiV86dOzNy9+th5UTvcFWYFj05J024CBlqNuc6aykCQ3aBIngAWw5NMl9coAfcBh1z64FXZO4KPjk2rPJIm8E2v/wpISjB0GZOf4Vr6rHBrggXGjkDX+YGF4bd+xCylS6K1hVKbvGlnYf1HBx6ewWp9zqkqyYyB9Vif7AGe/dH5jTzqPNYtt+q4fffxlFTEMvz0L/60szlWFrCfMKLAjA/YY24FvWBExJZ/SqFqugHS9zlhwlLbCoN/sKBG7JlJJIlQvGUxX5upgJI1QJ/RMWLWg+6gkUgk2xNhBMNWF1Ys8kvzBLFLhWrVOH7QknwObJkc3SJkMz5J/rIkt1VrwG0vAymiVUAfSLGTwvuwoMfA8tkAF7zQtMwWUwU5E2CDfh0grPBF/VPQGCKX2VO425Yuw/S2FokeV7bDRyQP3yPnusSZlTX45XyE8EkssK7j+lOZnl0bjbENZJEH0qlh2orRsC/xihBpo4M49G/Xae3lAlDBp9si7tzRCoyTo4v2KzW/r5rdmrqrfgrSFugpLYJpQsUZVM7d2vU9sJB5TTQ2tZuEAB8UAQprgNQF7IS1xkPafq321KYv21YQbPiqi6nBIzRzDbEEbl4qavS9MnfXgAC4z+RYQiv3XM6FRABsHJhwiK7UuE=
+
+notifications:
+  slack:
+    secure: gW5Z24CAFGHsCx7G6OexqcYsGL6vi+68X590Z4ISNpu5nM1sQVcbafcVKyswkH3cS5FQmcTlETrBk5mIekSQNKlPRi4oV8/DqXN9GUI8aoRV+sbsmH8S6B8KRT0MAzqJSGVJSpgQgbGhfba/BKx2dYz/km52+IVwB1/yPToANqeqbmwDffAsisW/I6gBh70tagRBumGCJ6tfRRUE0F6KNTxyiuB3WBvWC55mLCxz2A7ztk7/AkG3I44w909fULDfduMiW/dj/dc+sJm17SjfNyH7bjjWpUuYfsRw5xqVOBge0EIjo3E+9BGbfy43ajEdMRGwRuxfsPgxGcCR7MLFs2F98tyQs5fRXl9Un99LFLiIwMkwSwOJvhR1Oru7uRkEHKyNwPbtGk2CM1kwMkHJQr9xfgX63OuHkI5Srt2a4l6FZsG+jw9cc08c6UxAZGOayx7vu3fZjt84SqJuYrLyUtQxMVOPcoNftZlvdl06VZvk+Mdxu6Ry2bPgjI9rE94vWUZgp3NkAnkRH+upZXsQc5ixlK8DVYBBRNvvgf7gmN5N0HEoIqizGH7ISB9npomqxHrxxARBBzWwdL1ypwwAaEoaY8sxy18CyYkKh7XC9KlQBkAuiP3yzD0dLkBdWCTb1OcK7mU3WtGpCJWlM1MeO4ZoC5CbTPSPXtxm1fig2OA=
```

Comment:
that's cool. how do we use it?

Reply:
It's already working! just check the `travis-ci` channel on slack, we're already getting build notifications there. We can even disable email and just leave slack ones if we wanted